### DISCUSSION TITLE
V3.1.1 BLE Joystick doesn't work right

### DISCUSSION THREAD

**Discussion by zenlab3d** [ExpressLRS/ExpressLRS#1991] [General]
V3.1.1 (V3.1.0 also) BLE Joystick doesn't work right (in velocidrone simulator, radiomaster zorro elrs, win10)
V3.0.1 - BLE Joystick works fine
&
any versions ELRS BLE Joystick doesn't work under Android

**Comment by SilkjamFPV** [comment]
Yes!! Upgraded to 3.2.0 and can't fly in Velocidrone with it. Calibration doesn't work. Tried with 2 other modules.....they worked fine at 2.0 and after upgrading the stick movements I'm the Velocidrone Controller does not go all the way. Pretty much does not go in the negative.
